contrib/packaging/bootc.spec

```diff
@@ -115,13 +117,26 @@ sed -i -e '/https:\/\//d' cargo-vendor.txt
 %if %{with ostree_ext}
 make install-ostree-hooks DESTDIR=%{?buildroot}
 %endif
+
 mkdir -p %{buildroot}/%{dirname:%{system_reinstall_bootc_install_podman_path}}
 cat >%{?buildroot}/%{system_reinstall_bootc_install_podman_path} <<EOF
 #!/bin/bash
 exec dnf -y install podman
 EOF
 chmod +x %{?buildroot}/%{system_reinstall_bootc_install_podman_path}
 
+mkdir -p %{buildroot}/%{dirname:%{install_alongside_cleanup}}
+cat >%{?buildroot}/%{install_alongside_cleanup} <<EOF
+#!/bin/bash
```

Comment:
Can you put this in e.g. `contrib/` or so and have it installed by `make install`?

(And we want to detect the OS version in `make` (or `make install`) and only install these things if we're on a Fedora derivative; parse `/usr/lib/os-release` and check `ID_LIKE=fedora`)